<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormListTravel</class>
 <widget class="QWidget" name="FormListTravel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> List Travel</string>
  </property>
  <widget class="QLabel" name="imgSlot">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>301</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255, 170, 127);
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Твоя Картинка</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="reportEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>241</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelreport">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Буквально пару слов об этом месте :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>330</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Назвение места:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelrout">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Как до него добраться:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="routEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>330</y>
     <width>241</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="leftBtn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>370</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBatn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>370</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>370</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>edit and save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addImg">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>280</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>load picture</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
